README.md

```diff
@@ -37,38 +37,43 @@ brew install minio/stable/minio
 | Platform| Architecture | URL|
 | ----------| -------- | ------|
 |Apple macOS|64-bit Intel|https://dl.minio.io/server/minio/release/darwin-amd64/minio |
+|||https://dl.minio.io/server/minio/release/darwin-amd64/minio.asc |
 ```sh
-chmod 755 minio
-./minio server /data
+gpg2 --recv-keys 4405F3F0DDBA1B9E68A31D2512C74390F9AAC728
+gpg2 --verify minio.asc minio && chmod 755 minio && ./minio server /data
 ```
 
 ## GNU/Linux
 ### Binary Download
 | Platform| Architecture | URL|
 | ----------| -------- | ------|
 |GNU/Linux|64-bit Intel|https://dl.minio.io/server/minio/release/linux-amd64/minio |
+|||https://dl.minio.io/server/minio/release/linux-amd64/minio.asc |
 ```sh
-wget https://dl.minio.io/server/minio/release/linux-amd64/minio
-chmod +x minio
-./minio server /data
+wget https://dl.minio.io/server/minio/release/linux-amd64/minio{,.asc}
+gpg2 --recv-keys 4405F3F0DDBA1B9E68A31D2512C74390F9AAC728
+gpg2 --verify minio.asc minio && chmod 755 minio && ./minio server /data
 ```
 
 | Platform| Architecture | URL|
 | ----------| -------- | ------|
 |GNU/Linux|ppc64le|https://dl.minio.io/server/minio/release/linux-ppc64le/minio |
+|||https://dl.minio.io/server/minio/release/linux-ppc64le/minio.asc |
 ```sh
-wget https://dl.minio.io/server/minio/release/linux-ppc64le/minio
-chmod +x minio
-./minio server /data
+wget https://dl.minio.io/server/minio/release/linux-ppc64le/minio{,.asc}
```

Comment:
What is the need for , ?? . We don't create any file other than minio.asc

Reply:
In order to verify minio, we need to download the minio binary itself and the signature too. This command will download minio and minio.asc in the same command. This is known as brace expansion (for more info, refer to [the bash manual](https://www.gnu.org/software/bash/manual/html_node/Brace-Expansion.html)).

Do you prefer this form (a little bit cryptic but we can add comment), or should we execute two wget commands?

Reply:
Yes I know but let's not add such cryptic expansions, copy pastic with such things is prone to errors and we would like to keep it simple.

We should also mention verification separately, there is also sha256sum which should be mentioned.